app/components/icon_list/icon_list.html.erb

```diff
@@ -8,11 +8,13 @@
       
       <div class="IconList-text">
         <% if elem[:title] %>
-          <div class="IconList-title"> <%= elem[:title] %> </div>
+          <%= c 'heading', tag: :h3, class: 'IconList-title' do %>
+            <%= elem[:title] %>
+          <% end %>
```

Comment:
Actually, I’m not 100% sure if using headings inside of list items is valid and well-supported by assistive tech, I’ll check later. As an alternative, we could use a `strong` tag.

Reply:
Have you had a look already? 
From what I've found, using headings inside of list items seems to be valid because they are block-level elements. The only thing to watch out for is that the semantic structure is consistent for each list element, which is the case here. 
And in regards to assistive tech, I found a WCAG guideline to always use structural markup in a way that does represent relationships in the content. This is also the case here. 

I am not really sure where to refer to make sure that the actual assistive tech implementations support headings inside of list elements. If you have resources for that in mind, I would be happy for a pointer.

But so far, so good, and I am gonna merge now. :) 
Thank you for the review!

Reply:
> Have you had a look already?

No, but will do later and open a new PR in case smth needs to be fixed.

> I am not really sure where to refer to make sure that the actual assistive tech implementations support headings inside of list elements. If you have resources for that in mind, I would be happy for a pointer.

If unsure, I’m usually testing in VoiceOver on macOS, although that’s probably not the best way, as e.g. other screen readers for other platforms have higher market shares and might behaves differently.

Apart from that, I usually simply google or browse through blogs by authors who’ve written a lot on that topic.